se/__init__.py

```diff
@@ -196,14 +196,14 @@ def print_error(message: Union[SeException, str], verbose: bool = False) -> None
 	# is rendered in blue
 	message = regex.sub(r"`(.+?)`", stylize(r"\1", fg("light_blue")), message)
 
-	print("{}{} {}".format(MESSAGE_INDENT if verbose else "", stylize(" Error ", bg("red") + attr("bold")), message), file=sys.stderr)
+	print(f"{MESSAGE_INDENT if verbose else ''}{stylize(' Error ', bg('red') + attr('bold'))} {sys.stderr}")
 
 def print_warning(message: str, verbose: bool = False) -> None:
 	"""
 	Helper function to print a colored warning message to the console.
 	"""
 
-	print("{}{} {}".format(MESSAGE_INDENT if verbose else "", stylize(" Warning ", bg("yellow") + attr("bold")), message))
+	print(f"{MESSAGE_INDENT if verbose else ''} {stylize(' Warning ', bg('yellow') + attr('bold'))} {message}")
```

Comment:
I think we need to remove the space between the first `} {` to match the original formatting.

Reply:
This is done!